ldaptor/entry.py

```diff
@@ -16,16 +16,20 @@
 
 
 def sshaDigest(passphrase, salt=None):
+    """
+    Return the salted SHA for `passphrase` which is passed as bytes.
+    """
     if salt is None:
         salt = ''
         for i in range(8):
-            salt += chr(random.randint(0, 255))
+            salt += chr(random.randint(0, 127))
```

Comment:
this was changed to generate ascii characters... and make it easy to convert later to bytes.
I hope nobody is using this for production and this here just for demonstration purpose... i didn't wanted to spend more time with this